backend/apps/slack/management/commands/slack_sync_messages.py

```
@@ -0,0 +1,158 @@
+        
+        except KeyError as e:
+            logger.warning(f"Invalid message data, missing {e}: {message_data}")
+            return None
+
+    def _handle_slack_response(self, response, api_method):
+        """Handle Slack API response and raise exception if needed."""
+        if not response["ok"]:
+            error_message = f"{api_method} API call failed"
+            logger.error(error_message)
+            self.stdout.write(self.style.ERROR(error_message))
+            
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)